conformance/Makefile

```diff
@@ -22,9 +20,19 @@ create-kind-cluster: ## Create a kind cluster
 	kind export kubeconfig --kubeconfig $(KIND_KUBE_CONFIG_FOLDER)/config
 
 .PHONY: prepare-nkg
-prepare-nkg: ## Build and load NKG container on configured kind cluster
+prepare-nkg: ## Build and load NKG and NGINX containers on configured kind cluster
 	cd .. && make PREFIX=$(NKG_PREFIX) TAG=$(NKG_TAG) container
 	kind load docker-image $(NKG_PREFIX):$(NKG_TAG)
+	docker pull $(NGINX_IMAGE)
+	kind load docker-image $(NGINX_IMAGE)
+	yq -i 'with(.spec.template.spec.containers[0]; .image = "$(NKG_PREFIX):$(NKG_TAG)" | .imagePullPolicy = "Never")' ../deploy/manifests/deployment.yaml
+
+.PHONY: prepare-nkg-no-build
+prepare-nkg-no-build: ## Load NKG and NGINX containers on configured kind cluster
+	kind load docker-image $(NKG_PREFIX):$(NKG_TAG)
+	docker pull $(NGINX_IMAGE)
+	kind load docker-image $(NGINX_IMAGE)
+	yq -i 'with(.spec.template.spec.containers[0]; .image = "$(NKG_PREFIX):$(NKG_TAG)" | .imagePullPolicy = "Never")' ../deploy/manifests/deployment.yaml
```

Comment:
I don't think this is necessary here because this update must be performed before building the container.